torchvision/utils.py

```diff
@@ -223,6 +223,12 @@ def draw_bounding_boxes(
 
     colors = [(ImageColor.getrgb(color) if isinstance(color, str) else color) for color in colors]
 
+    if font is None and font_size is not None:
+        warnings.warn("Argument 'font_size' will be ignored since 'font' is not set.")
+        font_size = None
+    elif font is not None and font_size is None:
+        font_size = 10
```

Comment:
Following @oke-aditya's suggestion in https://github.com/pytorch/vision/pull/5785/files#r849794680. Please also remove L245.

```suggestion
    if font is None:
        if font_size is not None:
            warnings.warn("Argument 'font_size' will be ignored since 'font' is not set.")
        txt_font = ImageFont.load_default()
    else:
        txt_font = ImageFont.truetype(font=font, size=font_size or 10)
```

Reply:
I had same thing in my mind :smile: Makes the code more readable

Reply:
Done. Have updated the code accordingly.